prime-angular-frontend/src/app/modules/health-auth-site-reg/pages/site-information-page/site-information-form.model.ts

```diff
@@ -1,3 +1,3 @@
 import { HealthAuthoritySite } from '@health-auth/shared/models/health-authority-site.model';
 
-export interface SiteInformationForm extends Pick<HealthAuthoritySite, 'siteName' | 'pec' | 'securityGroupCode'> {}
+export interface SiteInformationForm extends Pick<HealthAuthoritySite, 'siteName' | 'pec' | 'securityGroupCode' | 'id'> { }
```

Comment:
Why is this id being added?

Reply:
Sorry i guess to explain more, since this is not a field on the form we should not have in on this model. I see later on you use this to get the site id. You might want to go another  route to get this id (since it is accessible in route.snapshot.params.sid)

Reply:
Sure I will try find a different way, it was kind of odd that I could see the `id` key in the object in the debugger but can't access it in code so I added it to the `Pick`, Site was null in that page and so I resorted to this, I'll see what else can provide a site ID.

Reply:
Fixed